src/test/scala/edu/ie3/mobsim/model/TripSimulationSpec.scala

```diff
@@ -20,7 +22,7 @@ import java.time.ZonedDateTime
 import javax.measure.quantity.Energy
 import scala.collection.mutable
 
-class TripSimulationSpec extends UnitSpec with ChargingBehaviorTestData {
+class TripSimulationSpec extends UnitSpec with IoUtilsTestData {
```

Comment:
Similar notes here compared to `ElectricVehicleSpec`:
- Assignments such as `val ev: ElectricVehicle = evAtChargingHub` are unnecessary
- Comparing `ev.getUuid` to itself does not make sense
- `destinationPoi` only needs to be checked once